models/model.js

```diff
@@ -3,3 +3,14 @@ const db = require("../db/connection");
 exports.selectTopics = () => {
   return db.query(`SELECT * FROM topics`);
 };
+
+exports.selectArticleByID = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id=$1`, [article_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "article does not exist" });
+      }
+      return rows;
```

Comment:
you should only send back the article to the controller so that the controller does not need to destructure anything